apps/admin/traefik2/traefik2.yaml

```diff
@@ -8,9 +8,13 @@ metadata:
 spec:
   releaseName: traefik
   chart:
-    repository: https://helm.traefik.io/traefik
-    name: traefik
-    version: 10.6.0
+    spec:
+      chart: ./
```

Comment:
is this right?

Reply:
think so https://github.com/traefik/traefik

Reply:
Where have you configured the helm chart location?

Reply:
added helm repo file